PatientWebApplication/HealthClinic.DAL/Service/RegularAppointmentService.cs

```diff
@@ -145,5 +153,62 @@ public Boolean isTermNotAvailable(DoctorUser doctor, TimeSpan time, String dateT
             return false;
         }
 
+        public List<DoctorAppointment> SimpleSearchAppointments(AppointmentReportSearchDto appointmentReportSearchDto)
+        {
+            List<DoctorAppointment> appointments = GetAppointmentsForPatient(appointmentReportSearchDto.PatientId);
+
+            appointments = SearchForDate(appointments, appointmentReportSearchDto);
+
+            appointments = SearchForDoctorNameAndSurname(appointments, appointmentReportSearchDto);
+
+            appointments = SearchForAppointmentType(appointments, appointmentReportSearchDto);
+
+            return appointments;
```

Comment:
Write in the form:
"return SearchForAppointmentType(SearchForDoctorNameAndSurname(SearchForDate(GetAppointmentsForPatient(appointmentReportSearchDto.PatientId), appointmentReportSearchDto), appointmentReportSearchDto), appointmentReportSearchDto);"